packages/react-ui-core/src/DateInput/__tests__/DateInput-test.js

```diff
@@ -1,20 +1,20 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import { mount } from 'enzyme'
 import DateInput from '../DateInput'
 
 describe('DateInput', () => {
   it('renders a DateInput', () => {
-    const wrapper = shallow(<DateInput />)
-    expect(wrapper.find('DatePicker')).toHaveLength(1)
+    const wrapper = mount(<DateInput />)
+    expect(wrapper.find('Datepicker')).toHaveLength(1)
   })
 
-  it('passes through props', () => {
```

Comment:
Can you add this test back in to test if passing in `unknownProp=unknownValue` gets passed through to `DatePicker`?

Reply:
I wouldn't expect it to, since the plugin itself doesn't spread props - they would get lost there

Reply:
In `render()` method you are passing `{...rest}` to the `DatePicker` component. So for example, if I wanted to set a DatePicker prop I should be able to pass it through this component.

Reply:
I guess I should remove rest, I suppose, and only pass through all the props that the  plugin expects